packages/machine-extractor/src/__tests__/parseResult.test.ts

```diff
@@ -130,109 +132,518 @@ describe('MachineParseResult', () => {
+          },
+          "name": "xstate.sendTo",
+        },
+        {
+          "expr": {
+            "actorRef": {
+              "type": "expression",
+              "value": "(ctx) => ctx.actorRef",
+            },
+            "event": {
+              "type": "expression",
+              "value": "() => ({type: 'event', id: 1})",
+            },
+            "options": {
+              "delay": undefined,
+              "id": undefined,
+            },
+          },
+          "name": "xstate.sendTo",
+        },
+        {
+          "expr": {
+            "actorRef": {
+              "type": "string",
+              "value": "actor",
+            },
+            "event": {
+              "type": {
+                "type": "string",
+                "value": "event",
+              },
+            },
+            "options": {
+              "delay": {
+                "type": "number",
+                "value": 2000,
+              },
+              "id": {
+                "type": "number",
+                "value": 2,
+              },
+            },
+          },
+          "name": "xstate.sendTo",
+        },
+        {
+          "expr": {
+            "actorRef": {
+              "type": "string",
+              "value": "actor",
+            },
+            "event": {
+              "type": {
+                "type": "string",
+                "value": "event",
+              },
+            },
+            "options": {
+              "delay": {
+                "type": "string",
+                "value": "namedDelay",
+              },
+              "id": {
+                "type": "string",
+                "value": "cancellationId",
+              },
+            },
+          },
+          "name": "xstate.sendTo",
+        },
+        {
+          "expr": {
+            "actorRef": {
+              "type": "string",
+              "value": "actor",
+            },
+            "event": {
+              "type": {
+                "type": "string",
+                "value": "event",
+              },
+            },
+            "options": {
+              "delay": {
+                "type": "expression",
+                "value": "delay: () => 'namedDelay'",
+              },
+              "id": undefined,
+            },
+          },
+          "name": "xstate.sendTo",
+        },
+        {
+          "expr": {
+            "actorRef": {
+              "type": "string",
+              "value": "actor",
+            },
+            "event": {
+              "type": {
+                "type": "string",
+                "value": "event",
+              },
+            },
+            "options": {
+              "delay": {
+                "type": "expression",
+                "value": "delay: function () {return 2000}",
+              },
+              "id": undefined,
+            },
+          },
+          "name": "xstate.sendTo",
+        },
+      ]
+    `);
+  });
+
+  it('Should extract stop expressions', () => {
```

Comment:
I'd prefer to have specific tests for those things, further down the road it's much better to see a failing test case title "extract stop action (string argument)" than to see "extract stop actions (all variants)". It also helps debugging failing test cases because it's easier to focus on a specific test case that might be failing, code paths within the implementation are hit less frequently etc